### PR TITLE
implement mock for landing page

### PR DESCRIPTION
**Pull request by BolajiOlajide**
#### What Does This PR Do?
Implement the mock for the landing page.

#### Description Of Task To Be Completed
Bring the mockup to life with HTML and CSS

#### Any Background Context You Want To Provide?
N/A

#### How can this be manually tested?
By starting the application and navigating to `localhost:8000`, you should see the screenshot below.

#### Screenshot(s)
screen shot 2017-11-20 at 14 52 05

#### What are the relevant github issues?
N/A

### CODE REVIEW

File: client/components/HomePage/LoginPage.js
Comment (ignatiusukwuoma) [inline client/components/HomePage/LoginPage.js]: There should be an `onClick={this.onSignIn}` to sign in the user

Comment (andela-sjames) [reply client/components/HomePage/LoginPage.js]: I suppose this should be done and tested as well

Comment (andela-sjames) [inline client/components/HomePage/LoginPage.js]: I suppose this should be done and tested as well

File: client/components/App.js
Comment (ignatiusukwuoma) [inline client/components/App.js]: Removing `<Header />` from the top level component just because you don't need it when one of its child components is active might not be a good idea since it's needed for other child components. 
How about checking the current url `this.props.location.pathname` within the Header component and if it matches `"/"`, return null instead of JSX?

File: client/assets/style.scss
Comment (ignatiusukwuoma) [inline client/assets/style.scss:45]: I like the design and layout but I think the `Sign In` button should be somewhere around the centre of the page, not at the bottom.

File: client/components/HomePage/LoginPage.js
Comment (ignatiusukwuoma) [inline client/components/HomePage/LoginPage.js]: I don't think `GET CLOSER TO YOUR SOCIAL MEETUP` correctly represents the purpose of Andela Socials. What about `STAY INFORMED OF EVENTS HAPPENING ACROSS ANDELA` or `SOCIALIZE MORE WITHIN ANDELA`... or something like that?

File: client/components/HomePage/LoginPage.js
Comment (BolajiOlajide) [reply client/components/HomePage/LoginPage.js]: Resolved. We haven't started writing tests yet.

**Review by ignatiusukwuoma — CHANGES_REQUESTED** [review]

File: client/components/App.js
Comment (BolajiOlajide) [reply client/components/App.js]: Don't know why I didn't think of this. Thanks @andela-iukwuoma 
Resolved.

File: client/components/HomePage/LoginPage.js
Comment (BolajiOlajide) [reply client/components/HomePage/LoginPage.js]: Resolved

File: client/assets/style.scss
Comment (BolajiOlajide) [reply client/assets/style.scss:45]: Well, the reason why it's at the bottom is to catch the interest of the user with featured meetups. We should have about 3 featured meetups in the next design but that should be the eye catcher of the user which will then lead you to sign in. 

That was my thought when designing the mock

Comment (ignatiusukwuoma) [reply client/assets/style.scss:45]: Nice idea, but the button can come be moved up.  We can work on that later though if someone else decide to take it up.

**Review by ignatiusukwuoma — APPROVED** [review]

**Review by ignatiusukwuoma — APPROVED** [review]
I think you should have made the changes requested on a different commit so we can easily see what was changed. I'll advise you do that on subsequent PRs

**Comment by BolajiOlajide** [comment]
Okay boss. No problem about that. @andela-iukwuoma 

@andela-sjames I'm waiting on your approval.

**Comment by BolajiOlajide** [comment]
screen shot 2017-11-20 at 22 54 55

Is this better @andela-iukwuoma

**Review by andela-sjames — APPROVED** [review]

**Pull request merged**
